oci/caps/utils.go

```diff
@@ -67,73 +68,104 @@ func GetAllCapabilities() []string {
 }
 
 // inSlice tests whether a string is contained in a slice of strings or not.
-// Comparison is case insensitive
+// Case sensitive comparisation is used for Capabilities field
```

Comment:
oh! I forgot to remove this part of the description

Reply:
Fixed :sunglasses: